docs/docs/modules/memory/examples/zep_memory.md

```diff
@@ -0,0 +1,156 @@
+# Zep Memory
```

Comment:
Can you convert this to an `.mdx` file? Helps with making sure examples are linted and formatted correctly:

https://github.com/hwchase17/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md#documentation-and-integration-tests

Reply:
Done

Reply:
It appears like API References that are auto-generated by docusaurus is not picking up class definition for zep_memory classes. Possibly because i am using hard path in the memory example imports. in my memory example under example/src/memory/zep.ts: I am unable to import {
  ZepChatMessageHistory,
  ZepMemory,
  ZepMemoryInput,
} from "**_langchain/memory/_**"

and instead have to use:
 from "**_../../../langchain/src/memory/zep_memory.js_**";

Side effect is that doc build is not able to autofind the class definition docs for zep_memory. 

Any ideas why this may be the case?

Reply:
Added a create_endpoint for memory (thanks for the tip @jacoblee93 I appear to have missed this for memory). Done